modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/DiagramBuilder.js

```diff
@@ -41,10 +41,18 @@ import populateNotificationsData from './util/populateNotificationsData';
 
 const deserializeUtil = new DeserializeUtil();
 
+function findEmptyElements(element, selectedLanguageId) {
+	if (element.data.label) {
+		if (!(selectedLanguageId in element.data.label)) {
```

Comment:
@M-IgorFranca I think you can just add the condition of the second `if` to the first `if`, placing a `&&` before it.

Reply:
yes it makes sense.